app/controllers/direct_assessments_controller.rb

```diff
@@ -1,36 +1,40 @@
 class DirectAssessmentsController < ApplicationController
   def new
-    @outcome = Outcome.find(params[:outcome_id])
-    @assessment = @outcome.direct_assessments.build
-    authorize(@outcome)
+    @assessment = DirectAssessmentForm.new(DirectAssessment.new, [])
```

Comment:
I think since the params are nested we'd need to do something like `params.fetch(:direct_assessment, {}).fetch(:outcome_ids, []), so I think passing the empty array is better. We could also have it as a default in the initializer.